test/unit/rpc/substrate/getBalance.test.ts

```diff
@@ -0,0 +1,58 @@
+import chai, {expect} from "chai";
+import sinonChai from "sinon-chai";
+import {MetamaskState} from "../../../../src/interfaces";
+import {WalletMock} from "../../crypto/wallet.mock.test";
+import {getBalance} from "../../../../src/rpc/substrate/getBalance";
+import ApiPromise from "@polkadot/api/promise";
+import { AccountInfo } from "@polkadot/types/interfaces/system";
+import sinon from "sinon";
+
+chai.use(sinonChai);
+
+describe('Test rpc handler function: getBalance', () => {
+
+  const walletStub = new WalletMock();
+
+  afterEach(function () {
+    walletStub.reset();
+  });
+
+  it('should return balance on saved keyring in state', async function () {
+    // wallet stub
+    walletStub.getPluginState.returns({
+      polkadot: {
+        account: {
+          keyring: {
+            address: "5Gk92fkWPUg6KNHSfP93UcPFhwGurM9RKAKU62Dg6upaCfH7",
+            // eslint-disable-next-line max-len
+            encoded: "0x3053020101300506032b6570042204206162613264643161313265656166646133666461363261613664666132316361cf043e13d9228d8a931ce4cc58efbd1ad6c5e2f1932c3174eb150dfaf9165b73a123032100cf043e13d9228d8a931ce4cc58efbd1ad6c5e2f1932c3174eb150dfaf9165b73",
+            encoding: {
+              content: ["pkcs8", "ed25519"],
+              type: "none",
+              version: "2"
+            },
+            meta: {}
+          },
+          seed: "aba2dd1a12eeafda3fda62aa6dfa21ca",
+        }
+      }
+    } as MetamaskState);
+    // api stub
+    const apiStub = {query: {system: {account: sinon.stub()}}};
```

Comment:
@mpetrunic I am not sure this is the best solution for stubbing API

Reply:
I think it's fine, it's hard to stub properties